index.js

```diff
@@ -135,7 +134,7 @@ function getSetSchemesData({
     });
 
     const dictatorArgs = Object.values({
-        avatar,
+        avatar: '0x0000000000000000000000000000000000000000',
         owner: '0xbBb06cD354D7f4e67677f090eCc3f6E5916E2447'
```

Comment:
lets make this as a Constant ..maybe in a common_constant param file (maybe json)

Reply:
I can just make it a constant in the file, why create a new file just for that?

Reply:
ok